src/common/telemetry/collect-telemetry/get-escape-hatches.ts

```diff
@@ -0,0 +1,25 @@
+import type {MaybePlural} from '../../MaybePlural'
+
+export type EscapeHatches = Record<string, MaybePlural<string>>
+
+export const getEscapeHatches = (source: string, methods: Array<string>): EscapeHatches => {
```

Comment:
@quesabe let's rename to `collectEscapeHatches` maybe? Would it still be relevant name? Also can you please add jsdoc?